notebooks/100_datamodules/102_mvtec.ipynb

```diff
@@ -58,10 +58,116 @@
    ]
```

Comment:
<p>Change "BeanTech" to MVTec</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/880/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>